js/validator.js

```diff
@@ -0,0 +1,127 @@
+function Validator(options){
+    var selectorRules={}
+    function Validate(inputElement,rule){
+        var errorElement= inputElement.parentElement.querySelector('.form-message');
```

Comment:
Để ý format code trong toàn file, em có thể bỏ ; cuối lệnh nhé
vd: errorElement = inputElement.parentElement.querySelector('.form-message')